src/main/webapp/app/entities/ttn/add-product-modal.tsx

```diff
@@ -219,6 +243,31 @@ export class AddProductModal extends React.Component<IAddProductModalProps, IAdd
                 <option value="REMOVED_FROM_STORAGE">REMOVED_FROM_STORAGE</option>
               </AvField>
             </AvGroup>
+            <AvGroup>
+              <Label for="rooms">
+                <span>Storage room</span>
+              </Label>
+              <AvField
+                name="rooms"
+                type="select"
+                onChange={this.handleRoomChange}
+                validate={{
+                  required: {
+                    value: true,
+                    errorMessage: translate('entity.validation.required')
+                  }
+                }}
+              >
+                <option defaultChecked />
+                {rooms
```

Comment:
It's not necessary to use ternary operation here, just use:
`{rooms && rooms.map(....)}`

Reply:
This is jHipster genrated code, ternary operations are used everywhere. We simply don't have enough time to fix all of them.